### ISSUE TITLE
Add wheel installation instructions to INSTALLING.md and ensure they work for model examples and demos

### ISSUE THREAD

**Issue by tt-rkim** [tenstorrent/tt-metal#9305]
cc: @mbahnasTT @dvartaniansTT @Shubhamsaboo 

Adding this as a requirement for WH readiness.

implement instructions here: https://github.com/tenstorrent/tt-metal/issues/12609#issuecomment-2354109453

**Comment by tt-rkim** [comment]
Waiting on customer team feedback

**Comment by mbahnasTT** [comment]
@tt-rkim thanks for the whl installation enablement. We'll test it this week on both GS and WH, and update you.
@dvartaniansTT @rbartonTT @jerryliangTT, would you please test the whl and we give back our feedback? Thanks!

**Comment by tt-rkim** [comment]
Thank you so much guys
Again please review the *whole* instructions again to make sure you have everything. There has been quite a few changes to things like `apt get` packages which may require you to re-run commands.

**Comment by tt-rkim** [comment]
Waiting for feedback from #12609

**Comment by tt-rkim** [comment]
From Dalar:

I confirm falcon7b model demo passes. and the WHL installation was smooth. but all models demos need to pass for an official WHL release as we see in the CI some model demos failed.

The detailed instructions added to the issue description such as following the installation guide from the whl branch/tag ... the note regarding env variables setting, ... should be added to the installation instructions for the customers as well.

From from my convo w/ @ksriv001 :

Yeah... this is one of those things where we want to make things as uniform and one-track as possible for users, but it's a piece of technical debt that we carried over from YEARS ago. I talked with the ttnn talking heads on this one, and we settled on treating the "production" environment as one which doesn't have ARCH_NAME or TT_METAL_HOME set
I actually came up with some ideas to try to see if I can add some better detection + error messaging to stop users of the wheel from setting too many env vars... let me know if you guys want that
